src/backend/joanie/core/utils/webhooks.py

```diff
@@ -79,3 +81,57 @@ def synchronize_course_runs(serialized_course_runs):
                     webhook["url"],
                     extra=extra,
                 )
+
+
+def synchronize_signed_document(request):
```

Comment:
belongs more to `views.py` than to utils?

Reply:
I can seen that there is webhook method to synchronize course runs on this file too.
At the top of the file, it's written :

"""
Utils that can be useful throughout Joanie's core app to synchronize data with other services
"""

I thought it would have been a good idea to add the synchronize_signed_document webhook method here as well. What do you think ?